partiql-parser/src/main/antlr/PartiQLParser.g4

```diff
@@ -358,7 +358,7 @@ excludeExprSteps
     ;
 
 fromClause
-    : FROM tableReference;
+    : FROM ( tableReference ( COMMA tableReference)* );
```

Comment:
Made some ANTLR grammar changes to account for the new modeling of `FROM` clauses to be better aligned with the SQL1999 BNF definition. See https://github.com/partiql/partiql-lang-kotlin/pull/1579#discussion_r1788110562 for some further context.